pkg/webhook/configuration/update.go

```diff
@@ -53,12 +53,6 @@ func Update(ctx context.Context, c client.Client, dc discovery.ServerResourcer,
 		return nil
 	}
 
-	// Ensure the scheme used by the client knows about ValidatingWebhookConfiguration.
```

Comment:
We can remove it here because the scheme has already been added in https://github.com/GoogleContainerTools/kpt-config-sync/blob/v1.16.0-rc.2/pkg/core/scheme.go#L87.

Suggest mentioning it in the commit message. Otherwise, it is not safe to delete without registering at least once.

Reply:
updated commit msg